python/tvm/tir/schedule/_type_checker.py

```diff
@@ -21,12 +21,17 @@
 import inspect
 from typing import Any, Callable, Dict, List, Optional, Tuple, TypeVar, Union
 import typing
+import sys
 
 
 def _is_none_type(type_: Any) -> bool:
     return type_ is None or type_ is type(None)
 
 
+def get_python_version():
```

Comment:
nit: feels like we can just do something like `if sys.version_info >= (3, 9):`

Reply:
This is mainly for the readability. Feel free to lemme know if you think this is overkill haha